recipe/meta.yaml

```diff
@@ -95,23 +93,21 @@ outputs:
         - pandas
         - pyarrow
         - python
+        # https://github.com/single-cell-data/TileDB-SOMA/issues/3444
         - scipy
```

Comment:
```suggestion
        - scipy <1.15.0
```

Reply:
@jdblischak thanks for catcthing the typo.

I still fear it won't let me pin under something that doesn't exist _yet_.

I'll spend another 45-minute run trying that ...